templates/template_code/TheSadRogue.GoRogueSadConsoleGame/StartingExample.csproj

```
@@ -2,14 +2,14 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="GoRogue" Version="2.6.0" />
     <PackageReference Include="MonoGame.Framework.DesktopGL.Core" Version="3.8.0.1" />
     <PackageReference Include="SadConsole" Version="8.9.1" />
```

Should change this up to 8.99.0

```suggestion
    <PackageReference Include="SadConsole" Version="8.99.0" />
```